.travis.yml

```diff
@@ -57,6 +57,7 @@ branches:
     - master
     - dev
     - /^(stable|release)\/.*$/
+    - /^(feature|fix)\/.*$/
```

Comment:
Why is this needed? We usually don't want to build branches _as well as_ Pull Requests